ios/FluentUI/Vnext/AvatarGroup/AvatarGroup.swift

```diff
@@ -125,36 +125,48 @@ public struct AvatarGroup: View {
         let ringOuterGap: CGFloat = tokens.ringOuterGap
         let ringOffset: CGFloat = tokens.ringThickness + tokens.ringInnerGap + tokens.ringOuterGap
         let x: CGFloat = size + tokens.interspace - tokens.ringThickness
-        HStack(spacing: 0) {
-            ForEach(0 ..< maxDisplayedAvatars, id: \.self) { index in
-                // If the avatar is part of Stack style and is not the last avatar in the sequence, create a cutout
-                let needsCutout = tokens.style == .stack && (overflowCount > 0 || index + 1 < maxDisplayedAvatars)
-                let avatar = avatars[index]
-                let currentAvatarHasRing = avatar.isRingVisible
-                let nextAvatarHasRing = index + 1 < maxDisplayedAvatars ? avatars[index + 1].isRingVisible : false
-
-                let ringPaddingInterspace = nextAvatarHasRing ? interspace - (ringOffset + ringOuterGap) : interspace - ringOffset
-                let noRingPaddingInterspace = nextAvatarHasRing ? interspace - ringOuterGap : interspace
-                let stackPadding = (currentAvatarHasRing ? ringPaddingInterspace : noRingPaddingInterspace)
-
-                let xOrigin = currentAvatarHasRing ? x + ringOffset : x
-                let yOrigin = currentAvatarHasRing ? (nextAvatarHasRing ? ringOuterGap : ringOffset) :
-                    (nextAvatarHasRing ? 0 - ringOffset + tokens.ringOuterGap : 0)
-                let cutoutSize = nextAvatarHasRing ? size + ringOffset + ringOuterGap : size
-
-                AvatarView(avatar)
-                    .modifyIf(needsCutout, { view in
-                        view.mask(AvatarCutout(xOrigin: xOrigin,
-                                               yOrigin: yOrigin,
-                                               cutoutSize: cutoutSize)
-                                    .fill(style: FillStyle(eoFill: true)))
-                    })
-                    .padding(.trailing, tokens.style == .stack ? stackPadding : interspace)
+        let avatarHeight = groupContainsRing() ? CGFloat(size + (ringOffset * 2)) : size
+        GeometryReader { _ in
```

Comment:
This seems to be related to the bug you found about iOS 13:

![Screen Shot 2021-07-23 at 10 32 13 AM](https://user-images.githubusercontent.com/68076145/126819806-0222d5eb-8007-48b7-9392-15fc81b15b50.png)

Reply:
Isn't the GeometryReader supposed to be used inside the HStack?
Weren't you supposed to get the geometry proxy to calculate the final x and y coordinates?

```
GeometryReader { geoProxy in
...
}
```